taqueria-vscode-extension/src/test/suite/extension.test.ts

```diff
@@ -5,102 +5,112 @@ import * as vscode from 'vscode';
 		});
-
-		// TODO: Need to figure out if it is possible to interact with showDialogBox
-		//      If no this test cannot be automated
-		//      I tried to use RobotJS, but it has an issue to work with parser properly
-		//      Even if I exclude it it keeps asking to rebuild it due to an old version of npm
-		//      https://stackoverflow.com/questions/46384591/node-was-compiled-against-a-different-node-js-version-using-node-module-versio
-
-		// await vscode.commands.executeCommand("taqueria.init").then( ()=>{
-		// });
-
-		// Example how to use sleep to stop VS Code for a moment
-		// await sleep(60000);
 	});
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/605
-	it.skip('Verify that VS Code command Taqueria Scaffold will scaffold a project', async () => {
-		// Open taquified project in workspace under testProjectDestination
+	it('Verify that VS Code command Taqueria Compile Ligo will compile Ligo contract', async () => {
+		// It creates another process
+		// https://stackoverflow.com/questions/51385812/is-there-a-way-to-open-a-workspace-from-an-extension-in-vs-code
+		// await vscodeMock.commands.executeCommand('vscode.openFolder', vscode.Uri.parse(testProjectDestination));
+		// await workspace.updateWorkspaceFolders(0, 1, { uri: Uri.parse()});
 
-		// Run taqueria scaffold command
-		await vscode.commands.executeCommand('taqueria.scaffold');
-	});
+		setOpenDialogMocks(`${ligoContractFileDestination}`);
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/686
-	it.skip('Verify that VS Code command Taqueria Start Sandbox starts flextesa sandbox', async () => {
-		// Install plugin
-		await vscode.commands.executeCommand('taqueria.install @taqueria/plugin-flextesa');
+		await fse.rm(testProjectDestination, { recursive: true });
+		await fse.mkdir(testProjectDestination, { recursive: true });
 
-		// Verify that
-		await vscode.commands.executeCommand('taqueria.start_sandbox local');
-	});
-
-	// TODO: https://github.com/ecadlabs/taqueria/issues/686
-	it.skip('Verify that VS Code command Taqueria List Accounts display accounts on running sandbox', async () => {
-		await vscode.commands.executeCommand('taqueria.list_accounts local');
-	});
+		await vscodeMock.commands.executeCommand('taqueria.init');
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/686
-	it.skip('Verify that VS Code command Taqueria Stop Sandbox stops flextesa sandbox', async () => {
-		await vscode.commands.executeCommand('taqueria.stop_sandbox local');
-	});
+		choosePlugin = '@taqueria/plugin-ligo';
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/645
-	it.skip('Verify that VS Code command Taqueria Compile Ligo will compile Ligo contract', async () => {
-		// Install plugin
-		await vscode.commands.executeCommand('taqueria.install @taqueria/plugin-ligo');
+		await vscodeMock.commands.executeCommand('taqueria.install');
 
-		await vscode.commands.executeCommand('taqueria.compile_ligo');
+		choosePlugin = '@taqueria/plugin-flextesa';
```

Comment:
Why are you now installing the flextesa plugin? I don't think this is needed for this test

Reply:
You're right, we just tested that multiple plugins could be installed. Removing this step.